shared/chat/conversation/attachment-fullscreen/index.desktop.tsx

```diff
@@ -41,6 +43,33 @@ const Fullscreen = React.memo(function Fullscreen(p: Props) {
     setIsZoomed(zoomed)
   }, [])
 
+  const [previewState, setPreviewState] = React.useState<PreviewState>(
+    path && previewPath && !isVideo ? 'loadingPreview' : 'cantDoFallback'
+  )
+
+  const onLoadError = React.useCallback(() => {
+    setPreviewState('cantDoFallback')
+  }, [])
+  const onLoadedFull = React.useCallback(() => {
+    setPreviewState('fullLoaded')
+  }, [])
+  const onLoadedPreview = React.useCallback(() => {
+    setPreviewState('loadingFull')
+    const img = new Image()
```

Comment:
on desktop we can load the image on a detached node and just swap and take advantage of the cache